website/docs/hardware-tools/three-point.md

```diff
@@ -0,0 +1,120 @@
+
+    <img src="https://github.com/farm-ng/amiga-dev-kit/assets/64480560/17460082-5160-40f3-ad39-97a8044e5a7e"
+     width="600"
+     height="200" />
+
+    <img src="https://github.com/farm-ng/amiga-dev-kit/assets/64480560/e051765c-3b6d-4058-8cbf-69742ecf35ba"
+     width="200"
+     height="190" />
+
+3. Mount the upper 3 point linkage to the 3point back plate.
+
+    <img src="https://github.com/farm-ng/amiga-dev-kit/assets/64480560/0d96efa8-735d-4670-a19a-8efb99629a05"
+     width="510"
+     height="200" />
+
+    <img src="https://github.com/farm-ng/amiga-dev-kit/assets/64480560/6df75a7a-d632-46e8-92c0-2c852139068d"
+     width="200"
+     height="200" />
+
+4. Secure the arm brackets to the 3 point backplate.
+
+     <img src="https://github.com/farm-ng/amiga-dev-kit/assets/64480560/079fd062-3fc6-4840-97af-34ca0d2e03e6"
+     width="500"
+     height="300" />
+
+5. Connect the upper 3 point linkage to the arm brackets with the adjustable lift arms.
+
+     <img src="https://github.com/farm-ng/amiga-dev-kit/assets/64480560/9a17f13c-6550-4ecc-918c-db76a4364955"
+     width="474"
+     height="200" />
+
+     <img src="https://github.com/farm-ng/amiga-dev-kit/assets/64480560/bc830964-4b00-49d9-93ab-343732594661"
+     width="200"
+     height="200" />
+
+6. Connect the turnbuckles to the arm links and the 3 point backplate.
+     <img src="https://github.com/farm-ng/amiga-dev-kit/assets/64480560/8282a4b6-6976-4fca-a39d-a7e040acaed9"
+     width="320"
+     height="200" />
+
+7. Connect the 3point cylinder actuator to back plate and the upper 3point linkage.
+     <img src="https://github.com/farm-ng/amiga-dev-kit/assets/64480560/451e1e25-c493-400f-9d7b-5c0009677ed3"
+     width="280"
+     height="200" />
+
+     <img src="https://github.com/farm-ng/amiga-dev-kit/assets/64480560/f091ba54-53f7-45df-b9d1-3eff3fbdbed5"
+     width="280"
+     height="200" />
+
+     <img src="https://github.com/farm-ng/amiga-dev-kit/assets/64480560/25db5975-db23-472d-8359-db72afbdb3eb"
+     width="280"
+     height="200" />
+
+8. Attach the adjustable top link to the 3 point backplate.
+     <img src="https://github.com/farm-ng/amiga-dev-kit/assets/64480560/277f2a9d-bdd1-4648-8061-231c078561fa"
+     width="320"
+     height="200" />
+
+9. Add T to the existing E-stop connection.
+     <img src="https://github.com/farm-ng/amiga-dev-kit/assets/64480560/82249dd7-bde9-45d4-855c-c78f890ea964"
+     width="320"
+     height="200" />
+
+10. Attach a CAN cable to the CAN hub.
+     <img src="https://github.com/farm-ng/amiga-dev-kit/assets/64480560/3e652b3a-9456-4ae1-a032-57fab9f0b86b"
+     width="320"
+     height="200" />
+11. Connect E-stop and CAN cables to H-bridge bringing power to it from the main harness.
+     <img src="https://github.com/farm-ng/amiga-dev-kit/assets/64480560/f511a27b-5d08-427e-bcf4-e42fc5444ba8"
+     width="500"
+     height="200" />
+
+12. Connect power to the 3point cylinder ( please note lower connection).
```

Comment:
Does that top connector just stay exposed? Seems dangerous / very not water proof

Reply:
that is the intended design right now. maybe will change in the future but as far as the instructions go it is accurate